addons/editor-devtools/DevTools.js

```diff
@@ -1622,6 +1631,10 @@ export default class DevTools {
       return;
     }
 
+    if (typeof this.addon.settings !== undefined && !this.addon.settings.get("enableBlockInjector")) {
```

Comment:
"enableBlockInjector" doesn't seem to be a setting.

Reply:
Indeed - These flags don't exist yet - WL said to hang back on these so they will default to enabled for the time-being until key bindings are configurable...

Reply:
@griffpatch This check you are doing here always returns true because undefined needs to be a string. Try `"undefined" !== undefined` in the console.
![image](https://user-images.githubusercontent.com/72760579/114630361-69d53d80-9c88-11eb-9629-8477e5f728c1.png)

Reply:
but it shouldn't ever be undefined? or am I missing something

Reply:
Yes, it is undefined when running the addon through the extension. Remember, the extension and the addon both use the same code.

Reply:
ah, so on the extension, all features are always enabled?

Reply:
> Yes, it is undefined when running the addon through the extension. Remember, the extension and the addon both use the same code.

Hmm, I just stripped these one - Are they required after all (only with quotes). I was assuming that perhaps they weren't needed anymore.

Reply:
> Hmm, I just stripped these one - Are they required after all (only with quotes). I was assuming that perhaps they weren't needed anymore.

I would assume that they would be needed because the devtools extension does not support setting, but unless we really wanted to add them, they could be in editor settings, or maybe settings in the browser popup. In the extension, it should be erroring out saying "Cannot read property 'get' of undefined".

Reply:
W_L reminded me that there is an alternative, by checking `addon.self._isDevtoolsExtension` [which is true for the devtools extension](https://github.com/ScratchAddons/DevtoolsExtension/blob/master/inject/run-addon.js#L47).